main/lsp/LSPTypechecker.cc

```diff
@@ -80,13 +80,17 @@ void LSPTypechecker::initialize(LSPFileUpdates updates, WorkerPool &workers) {
     indexed = move(updates.updatedFileIndexes);
     // Initialization typecheck is not cancelable.
     // TODO(jvilk): Make it preemptible.
+    errorReporter.beginEpoch(updates.epoch, {});
     auto committed = runSlowPath(move(updates), workers, /* cancelable */ false);
+    errorReporter.endEpoch(updates.epoch);
```

Comment:
`updates` gets moved on the line above. I'm not sure if this will still retain the same epoch value.